src/Microsoft.Graph/Requests/Generated/UserDrivesCollectionResponse.cs

```diff
@@ -13,13 +13,13 @@ namespace Microsoft.Graph
     /// <summary>
     /// The type UserDrivesCollectionResponse.
     /// </summary>
-    [DataContract]
+    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
     public class UserDrivesCollectionResponse
     {
         /// <summary>
         /// Gets or sets the <see cref="IUserDrivesCollectionPage"/> value.
         /// </summary>
-        [DataMember(Name = "value", EmitDefaultValue = false, IsRequired = false)]
+		[JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName ="value", Required = Required.Default)]
```

Comment:
thanks for the explanation, based on we are in the driection moving to GraphAPI, I will not do the upgrade this time except if users get issues with the standard.